ocs_ci/ocs/ui/page_objects/storage_clients.py

```diff
@@ -12,15 +14,33 @@ class StorageClients(BaseUI):
 
     def __init__(self):
         super().__init__()
+        self.ocs_version = version.get_semantic_ocs_version_from_config()
 
-    def generate_client_onboarding_ticket(self):
+    def generate_client_onboarding_ticket_ui(self, storage_quota=None):
         """
         Generate a client onboarding ticket
 
         Returns:
             str: onboarding_key
         """
         self.do_click(self.storage_clients_loc["generate_client_onboarding_ticket"])
+        ValidationUI().verify_storage_clients_page()
```

Comment:
I know you merged it before, but is it possible to move the method verify_storage_clients_page to this class? It really belongs to StorageClients, so you would not create ValidationUI instance here but use self. 
That will save us some time because ValidationUI constructor loads all the parents up to BaseUI and checks the PageNavigator presence